modular_skyrat/modules/contractor/code/items/baton.dm

```diff
@@ -52,24 +69,36 @@
 
 /obj/item/melee/baton/telescopic/contractor_baton/additional_effects_non_cyborg(mob/living/carbon/target, mob/living/user)
 	. = ..()
-	if(!((upgrade_flags & BATON_MUTE_UPGRADE) && !(upgrade_flags & BATON_ALL_UPGRADE))|| !istype(target))
+	if(!istype(target))
 		return
-	if(target.silent >= (MUTE_CYCLES * 2))
-		return
-	target.silent += MUTE_CYCLES //10 seconds of mute a hit up to 20 max
+	if((upgrade_flags & BATON_MUTE_UPGRADE) || (upgrade_flags & BATON_ALL_UPGRADE))
+		if(target.silent < (MUTE_CYCLES * MUTE_MAX_MOD))
+			target.silent += MUTE_CYCLES //10 seconds of mute a hit up to 20 max
+	if((upgrade_flags & BATON_FOCUS_UPGRADE) || (upgrade_flags & BATON_ALL_UPGRADE))
+		if(target == user.mind?.opposing_force?.contractor_hub?.current_contract?.contract.target?.current) // Pain
+			target.apply_damage(BONUS_STAMINA_DAM, STAMINA, BODY_ZONE_CHEST)
+			target.stuttering += BONUS_STUTTER
 
 /obj/item/melee/baton/telescopic/contractor_baton/upgraded
 	upgrade_flags = BATON_ALL_UPGRADE
 	desc = "A compact, specialised baton assigned to Syndicate contractors. Applies light electrical shocks to targets. This one seems to be fully upgraded with unremovable parts."
 
+/obj/item/melee/baton/telescopic/contractor_baton/upgraded/Initialize(mapload)
+	. = ..()
+	for(var/num in 1 to CUFF_MAXIMUM)
```

Comment:
Something that's usually done is to keep that variable to either `var/i` or `var/_` to show that it's not actually used anywhere, especially the latter, and just used for counting purposes.